constrained-voucher.mkd

```diff
@@ -572,6 +572,12 @@ Knowing the overall progress of the data transfer may be helpful in certain case
 
 ## Registrar Extensions {#brski-extensions-registrar}
 
+Before a Registrar forwards a COSE-signed voucher from MASA to the Pledge, it MUST remove any "x5bag" or "x5chain"
+unprotected COSE header attributes (which are defined in {{RFC9360}}).
+The contents of these unprotected attributes are solely for validation/logging use by the Registrar.
+Removing these attributes reduces the voucher size on the constrained network path back to the Pledge.
+Note that any attributes in the COSE protected header are left untouched by this removal step.
```

Comment:
This sentence seems more confusing than helpful until the reader realizes protected!=unprotected.  Maybe omit this sentence?

Reply:
Yes, can be removed!